docs/pages/code/operations/cache.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use loophp\collection\Collection;
+
+include __DIR__ . '/../../../../vendor/autoload.php';
+
+$fopen = fopen(__DIR__ . '/vendor/autoload.php', 'rb');
+
+$collection = Collection::fromResource($fopen)
+    ->cache();
+
+for ($i = 0; 3 > $i; ++$i) {
+    foreach ($collection as $value) {
+        print_r($value);
+    }
+}
+
```

Comment:
can we pls add some comments to show what will be printed in each of these cases?

Reply:
Yes, will do